ceph/cephfs/README.md

```diff
@@ -16,16 +16,16 @@ docker build -t cephfs-provisioner .
 * Create a Ceph admin secret
 
 ```bash
-ceph auth get client.admin 2>&1 |grep "key = " |a^C '{print  $3'} |xargs echo -n > /tmp/secret
+ceph auth get client.admin 2>&1 |grep "key = " |awk '{print  $3'} |xargs echo -n > /tmp/secret
 kubectl create secret generic ceph-secret-admin --from-file=/tmp/secret --namespace=kube-system
 ```
 
 * Start CephFS provisioner
 
-Assume kubeconfig is at `/root/.kube`:
+The following example uses `cephfs-provisioner-1` as the identity for the instance and assumes kubeconfig is at `/root/.kube`. If there are multiple provisioners, each should have a different identity.
```

Comment:
it's also needed if the pod gets restarted, otherwise the 'new' pod wont delete the 'old' pod's storage to be safe since the identities mismatch. But maybe that's okay, the user just needs to know that he is then responsible for deleting the 'old' pod's storage. Does ceph fs provisioner need to be stateful at all? Would somebody need to run more than one? 

(I am struggling still with how to communicate this stuff to users :) )